packages/azure-openapi-validator/autorest/src/spectral-plugin-func.ts

```diff
@@ -225,7 +213,9 @@ async function runSpectral(
   return mergedResults
 }
 
-export async function getRuleSet(openapiType: OpenApiTypes): Promise<[ruleset: Ruleset, ruleSetIncludingRPCGuidelineCode: any, namesOfRulesInStagingOnly: string[]]> {
+export async function getRuleSet(
+  openapiType: OpenApiTypes
+): Promise<[ruleset: Ruleset, c: any, namesOfRulesInStagingOnly: string[]]> {
```

Comment:
>c [](http://example.com/codeflow?start=30&length=1)

Can we have a descriptive variable name

Reply:
Oh wow sorry for that, that was not intended :D